docs/Configuration/AdvancedMode.md

```diff
@@ -325,6 +326,32 @@ Below is an example for collecting a bitbucket repo.
 - `owner`: the owner of the repository.
 - `repo`: the bitbucket repository name.
 
+### 10. SonarQube
+
+Below is an example for collecting a SonarQube project.
+
+```json
+[
+  [
+    {
+      "plugin": "sonarqube",
+      "options": {
+        "connectionId": 1,
+        "projectKey": "testDevLake"
+      }
+    }
+  ]
+]
+```
+
+- `connectionId`: The ID of your SonarQube connection at page http://localhost:4000/connections/sonarqube.
+- `projectKey`: the project name of the SonarQube. To find the project key in SonarQube, you can follow these steps:
```

Comment:
The project key of the SonarQube. To find the project key in SonarQube, please follow the steps: